apps/ui/pages/reservation-unit/[id].tsx

```diff
@@ -476,7 +468,7 @@ const ReservationUnit = ({
       searchUIDate != null && isValidDate(searchUIDate)
         ? searchDate ?? ""
         : defaultDateString,
-    duration: searchDuration ?? minReservationDuration,
+    duration: searchDuration ?? minReservationDurationMinutes,
```

Comment:
Should this allow duration to be less than minimum duration?
e.g. User searched for 1 hour but minimum duration is 2 hours, then the duration would be 1h

Simple fix would be something like: `duration: Math.max(searchDuration ?? 0, minReservationDurationMinutes)`